src/Views/DiskView.vala

```diff
@@ -142,21 +144,26 @@ public class Installer.DiskView : AbstractInstallerView {
                 label + " (" + path + ")",
                 icon_name,
                 path,
-                disk.get_sectors () * disk.get_sector_size ()
+                size
             );
 
             disk_grid.add (disk_button);
-            disk_button.clicked.connect (() => {
-                if (disk_button.active) {
-                    disk_grid.get_children ().foreach ((child) => {
-                        ((Gtk.ToggleButton)child).active = child == disk_button;
-                    });
 
-                    next_button.sensitive = true;
-                } else {
-                    next_button.sensitive = false;
-                }
-            });
+            if (size < 5000000000) {
```

Comment:
This should probably be a configuration option, or a value read from the `/cdrom/casper/filesystem.size` file

Reply:
@jackpot51 Think I should implement that logic in distinst?